lib/timetable-form/FormStage.tsx

```diff
@@ -0,0 +1,93 @@
+import { ReactNode, useState } from "react";
+
+import { EventKey } from "../constants";
+import { Button, DateInput, ErrorSummary, TextArea } from "../gds-components";
+import { useValidation } from "./useValidation";
+
+import styles from "./styles.module.css";
+import "govuk-frontend/dist/govuk/govuk-frontend.min.css";
```

Comment:
We won't need to import this here when it's being used in the form, so maybe this should be moved to the story?